pygmt/tests/test_grdcut.py

```diff
@@ -5,11 +5,18 @@
 import numpy as np
 import pytest
 import xarray as xr
-from pygmt import grdcut, load_dataarray
+from pygmt import grdcut, load_dataarray, which
 from pygmt.exceptions import GMTInvalidInput
 from pygmt.helpers import GMTTempFile
 from pygmt.helpers.testing import load_static_earth_relief
 
+try:
+    import rioxarray
+
+    _HAS_RIOXARRAY = True
+except ImportError:
+    _HAS_RIOXARRAY = False
+
```

Comment:
Maybe best to move the image specific tests to a `test_grdcut_image.py` file (similar to what we have for `test_grdimage_image.py`). Then it's possible to use `rioxarray = pytest.importorskip("rioxarray")`.

Reply:
> Maybe best to move the image specific tests to a `test_grdcut_image.py` file (similar to what we have for `test_grdimage_image.py`). 

I'm OK with moving the tests to a separate file.

> Then it's possible to use `rioxarray = pytest.importorskip("rioxarray")`.

Currently yes, but after #3128 is done, rioxarray is no longer needed for the `test_grdcut_image_file` test, then we can't use `pytest.importorskip("rioxarray")` to skip all the tests.

Reply:
Ok, sounds good.